### PR TITLE
Developer profile branch

### PR DESCRIPTION
**Pull request by monishkajha17**
PR request for issue no. #194

### CODE REVIEW

**Review by Rishikavishnoi — CHANGES_REQUESTED** [review]
@monishkajha17 add a screenshot of your project in the README.md file

**Comment by monishkajha17** [comment]
@Rishikavishnoi okay done please check

**Comment by monishkajha17** [comment]
Is this PR is ready to merge or it still have problems. Kindly confirm

**Review by Vanshiii203 — APPROVED** [review]

**Comment by monishkajha17** [comment]
When this PR will be merged?

File: Projects/Developer  Profile card/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Developer Porfile Card</title>
+</head>
+<body>
+    <div class="profile_card">
+        <div class="user_image">
+            <img src="C:\Users\Monishka jha\Desktop\WhatsApp Image 2022-11-21 at 00.55.42.jpg" height="100" width="100" alt="profile">
```
Comment (TusharKesarwani) [inline Projects/Developer  Profile card/index.html:13]: Add required files and change the link according to that file

**Comment by monishkajha17** [comment]
@TusharKesarwani Pls check

**Comment by TusharKesarwani** [comment]
Screenshot
@monishkajha17 
The image is not visible

**Comment by monishkajha17** [comment]
@TusharKesarwani  Can I just make the image into link because its not working in folder

**Comment by monishkajha17** [comment]
@TusharKesarwani I just updated a image kindly check

**Comment by TusharKesarwani** [comment]
Screenshot
Icons are still not working
@monishkajha17

**Comment by monishkajha17** [comment]
@TusharKesarwani  Kindly check now I have fixed the icons

**Review by TusharKesarwani — APPROVED** [review]

**Pull request merged**